vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/DragAndDropGridIT.java

```diff
@@ -36,6 +43,12 @@ public void init() {
         grid = $(GridElement.class).first();
     }
 
+    @Test
+    public void dragAndDropDefined_gridLoaded_noErrors() {
+        List<LogEntry> logs = getLogEntries(Level.SEVERE);
+        Assert.assertTrue(logs.isEmpty());
```

Comment:
```suggestion
        checkLogsForErrors();
```
We usually use this to check for client-side errors.

Reply:
Ah, wasn't aware of this helper method. Done.